src/site/design-elements/buttons/index.html

```diff
@@ -1,23 +1,27 @@
 <div id="buttons" class="headroom-large">
-  <h5>Buttons</h5>
+  <h2>Buttons</h2>
   <p>This is the primary button style for important calls to action. Two primary buttons cannot be used next to each other.</p>
   <div data-xrayhtml>
-    <button class="btn btn-mapzen">main call to action</button>
+    <button class="btn btn-darkpurple">primary button</button>
```

Comment:
ok, so i changed that to btn-mapzen because buttons.scss in our styleguide has an override for our newest styles. There is still btn-mapzen-alt, btn-darkpurple, btn-white are in there as new styles. @rfriberg do you want to change that and then I can reflect that in the wiki?
I'm sure we are already referencing those names on the website so I'm scared to change the name and break stuff.